package-lock.json

```diff
@@ -0,0 +1,632 @@
+{
+  "name": "rails_api_base",
```

Comment:
you don't have to commit this file, we use `yarn` not `npm`

Reply:
Ok no problem